packages/ffe-core-react/src/typography/Heading.js

```diff
@@ -28,6 +29,7 @@ const makeHeading = level => {
                     { [`${headingClass}--inline`]: inline },
                     { [`${headingClass}--no-margin`]: noMargin },
                     { [`${headingClass}--with-border`]: withBorder },
+                    { [`${headingClass}--text-center`]: textCenter },
```

Comment:
Can we please rename this to `centerText` like for [the grid](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-grid-react/src/GridCol.js#L41) (or rename the grid modifier to `textCenter`, I don't really care as long as it's consistent).

Reply:
I am now consistent with Paragraph that also uses "textCenter". I though that would make more sense because they are both text element.

Reply:
If we're already inconsistent across modules I guess it doesn't really matter. Hopefully at some point we'll have a dedicated team that can also work to keep the various APIs predictable :)